projects/angular/src/forms/common/STYLES.label.md

```diff
@@ -0,0 +1,48 @@
+# Label
+
+## Variables
+
+| Variable Name                   | Description                            |
+| ------------------------------- | -------------------------------------- |
+| $clr-label-padding-top-bottom   | Padding top and bottom for labels      |
```

Comment:
Document `--clr-label-padding-top-bottom` instead. The description is fine.
Please apply for all other items in the list, just make sure, that such a property exists after the rename.